docs/blog/2019-03-18-releasing-new-schema-customization/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: New Schema Customization API - Available in Gatsby 2.2.0
+date: 2019-03-18
+author: Mikhail Novikov
+tags:
+  - schema
+  - graphql
+---
+
+Two weeks ago, we announced our plans for a [new schema customization API](/blog/2019-03-04-new-schema-customization/). Today we are making this set of new APIs and enhancements available to all in `gatsby`@`2.2.0`.
+
+```shell
+npm install gatsby --save
+```
+
+# Recap of schema customization
+
+Before this change, the Gatsby GraphQL schema was generated automatically from the data that the user added to Gatsby. While very convenient and easy to start, changes to the data could cause changes to the schema, which could cause breakage in unrelated locations. Those bugs were confusing and hard to debug. To alleviate this problem, we've added a schema customization API that lets you customize, fix, and enhance types in your Gatsby GraphQL schema.
+
+There are two new APIs, `createTypes` and `createResolvers`.
+
+## `createTypes`
+
+`createTypes` can be used to define or fix your Node types GraphQL representation.
```

Comment:
Can you add a few words explaining what "Node types GraphQL representation" means in case people don't already know?

Reply:
I've rephrased to 

```md
`createTypes` can be used to define, fix, or extend a Gatsby GraphQL type that represents some of your app's data.
```